setup.py

```diff
@@ -262,6 +262,7 @@ def run(self):
     entry_points={
         'console_scripts': [
             'borg = borg.archiver:main',
+            'borgmount = borg.archiver:main_mount',
```

Comment:
other specific mount commands are named mount.xxx (e.g. mount.cifs), does it make sense to follow that convention?